internal/metrics/metricsManager.go

```diff
@@ -48,6 +48,7 @@ const (
 	namespace                  = ""
 	subsystem                  = "service"
 	openshiftVersionLabel      = "openshiftVersion"
+	clusterID                  = "clusterId"
```

Comment:
please change the name to clusterIdLabel - it's a little bit confusing with the clusterId parameter.

Reply:
ack